src/SciAssist/utils/windows_pdf2text.py

```diff
@@ -19,7 +19,8 @@ def process_pdf(path):
     for page_layout in extract_pages(path):
         for element in page_layout:
             if "get_text" in dir(element):
-                text = element.get_text().replace("\n","")
+                text = element.get_text().replace("-\n","")
+                text = text.replace("\n"," ")
```

Comment:
concatenate breaking lines. But what if this is a normal paragraph break?

Reply:
If normal paragraph break, "\n" will be replaced by a space " ". Is this not the intended result?

Reply:
It is the desired.